### ISSUE TITLE
No STOP when detecting wrong collimator material

### ISSUE THREAD

**Issue by kyrsjo** [SixTrack/SixTrack#29]
In the subroutine "collimate2", a list of materials is accepted:
```
      if (c_material.eq.'BE') then
         mat = 1
      elseif (c_material.eq.'AL') then
         mat = 2
      elseif (c_material.eq.'CU') then
         mat = 3
      elseif (c_material.eq.'W') then
         mat = 4
      elseif (c_material.eq.'PB') then
         mat = 5
      elseif (c_material.eq.'C') then
         mat = 6
      elseif (c_material.eq.'C2') then
         mat = 7
!02/2008 TW added vacuum and black absorber (was missing) 
      elseif (c_material.eq.'VA') then
         mat = 11
      elseif (c_material.eq.'BL') then
         mat = 12
      else
+if cr
         write(lout,*) 'ERR>  Material not found. STOP', c_material
+ei
+if .not.cr
         write(*,*) 'ERR>  Material not found. STOP', c_material
+ei
!        STOP
      endif
```

However, if the material is not recognized, the program just prints a warning and the continues, since the STOP command has been commented out.

Are there any reason to not reinstate this STOP? Quite a few CPU hours were recently wasted due to a bad CollDB which contained an unrecognized material...

**Comment by kyrsjo** [comment]
@rbruce1

**Comment by rbruce1** [comment]
Hi Kyrre,
no, I wasn't aware of this and I don't see any reason why it should be 
commented out.

Cheers,
Roderik

**Comment by kyrsjo** [comment]
I could make a patch which fixes this and also #6.

**Comment by kyrsjo** [comment]
TODO: Patch which fixes #29 #6 #30 #31 . Schedule: After finishing testing for #11

**Comment by kyrsjo** [comment]
Fixed in https://github.com/kyrsjo/SixTrack/commit/b705131c8f1eef98733f0a8ceac9c322f4338cb1
